src/singular.js

```diff
@@ -11,6 +11,13 @@ function createIframe(name, src) {
     return iframe;
 }
 
+function afterDomContentLoaded(opIframe, rpIframe) {
```

Comment:
The naming of this function is specific to the use case rather than the functionality. Could the naming be changed to indicate what the function does?

Reply:
We will rename the function to `appendFramesToBody`.